index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <main id="main-doc">
+
+    <div id="left-part">
+      <nav id="navbar">
+        <header>
+          <h1>Java Documentation</h1>
+        </header>
+        <ul id="menu-list">
+          <hr />
+          <li><a href="#introduction" class="nav-link">Introduction</a></li>
+          <hr />
+          <li><a href="#trademark" class="nav-link">Trademark</a></li>
+          <hr />
+          <li><a href="#features" class="nav-link">Features</a></li>
+          <hr />
+          <li><a href="#syntax" class="nav-link">Syntax</a></li>
+          <hr />
+          <li><a href="#development_tools" class="nav-link">Development Tools</a></li>
+          <hr />
+        </ul>
+      </nav>
+    </div>
```

Comment:
Use proper semantics. The best practice is to use the section tag for cleaner readable code.